book/P2C10_rules.qmd

```diff
@@ -20,6 +20,8 @@ Before delving into these measures, we will first describe scoring rules in the
 Scoring rules are pointwise losses, which means a loss is calculated for all observations and the sample mean is taken as the final score.
 To simplify notation, we only discuss scoring rules in the context of a single observation where $L_i(\hatS_i, t_i, \delta_i)$ would be the loss calculated for some observation $i$ where $\hatS_i$ is the predicted survival function (from which other distribution functions can be derived), and $(t_i, \delta_i)$ is the observed survival outcome.
 
+Note that unlike other chapters, there is not a separate section for 'beyond right-censoring' here as the primary results all fit within @sec-logloss.
+
```

Comment:
Comments for below: I think the bit about constant predictoin being strictly proper but meaningless is confusing and quite an edge case that no practitioneer/reader of the book would be concerned with. They want to know from us what the should use, not some philosophical discussion. So I think it could be safely deleted without loss.